docs/table-optimization.md

```diff
@@ -12,22 +12,26 @@ want to use them in production.
 ## Example
 
 ```java
-
 FluoConfiguration fluoConf = ...
 
-//Post initialization table optimizations
-Pirtos pirtos = new Pirtos();
-
 //export queue configure method will return table optimizations it would like made
-pirtos.merge(ExportQueue.configure(fluoConf, ...));
+ExportQueue.configure(fluoConf, ...);
 
 //CollisionFreeMap.configure() will return table optimizations it would like made
-pritos.merge(CollisionFreeMap.configure(fluoConf, ...));
+CollisionFreeMap.configure(fluoConf, ...);
 
 //initialize Fluo
 FluoFactory.newAdmin(fluoConf).initialize(...)
 
-//perform table optimizations
-TableOperations.optimizeTable(fluoConf, pirtos);
-
+//Automatically optimize the Fluo table for the two recipes configured before
+//initialization
```

Comment:
I actually did intend to say before.  I suppose that wording is confusing.  I can simplify it to `Automatically optimize the Fluo table for all configured recipes`